scripts/tests/chiptest/__init__.py

```diff
@@ -164,6 +164,8 @@ def _GetChipToolUnsupportedTests() -> Set[str]:
     """Tests that fail in chip-tool for some reason"""
     return {
         "TestDiagnosticLogsDownloadCommand",  # chip-tool does not implement a bdx download command.
+        "Test_TC_TMP_2_1",  # Test incorrectly implemented as it relies on saving value that may be null, which maybe
```

Comment:
If the test is being disabled _everywhere_, shouldn't it be moved out of CI to manual tests instead or something?  Otherwise we are left with the very incorrect impression that it actually runs in CI.

But also "may be" instead of "maybe", and the problem is not saving a value that may be null, it's doing arithmetic on a value that may be null, no?  And we should be able to fix the test to not do those checks if the value is null....  Followup for that, presumably.

Reply:
https://github.com/project-chip/connectedhomeip/pull/32881 Actually fixes the test.

I will wait for it to merge and merge mine afterwards. That way there will be no changed to this file or manualTests.json

Keeping it in for the time being so that I can confirm CI passes